templates/provider/kcm/templates/rbac/controller/roles.yaml

```diff
@@ -262,6 +263,26 @@ rules:
   - deployments
   verbs:
   - list
+- apiGroups:
```

Comment:
unrelated to the line: what are user-facing permissions? i believe they should also be added, please refer to the `templates/provider/kcm/templates/rbac/user-facing`

Reply:
Not sure I understand re: the packaging. So the controllers will remain in the controllers package correct? Then what does in the "ipam" package? The types?